docs/examples/redshift.rst

```diff
@@ -0,0 +1,66 @@
+Redshift Example Project
```

Comment:
"Redshift Estimation" is probably better ("example project" is a bit of a given).